Backpack/Carousel/Classes/CarouselContentViewController.swift

```diff
@@ -0,0 +1,46 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+final class CarouselContentViewController: UIViewController {
+    
+    private var carouselView: UIView = {
+        let view = UIView()
+        view.clipsToBounds = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.clipsToBounds = true
+        view.addSubview(carouselView)
+        NSLayoutConstraint.activate([
+            carouselView.topAnchor.constraint(equalTo: view.topAnchor),
+            carouselView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            carouselView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            carouselView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+    
+    func render(view: UIView) {
+        view.translatesAutoresizingMaskIntoConstraints = false
+        carouselView = view
+    }
+}
```

Comment:
I think a better implementation for this would be an initialiser that receives the view instead of having a `render` method.
The problem with this is that it introduces a tiny bit of `temporal coupling`, where you need to remember to call render after creating an instance, and know not to call it after the fact. To avoid all this, lets make the contained view a required paramter of this view controller's initialiser